packages/app/pages/api/me.ts

```diff
@@ -1,12 +1,29 @@
 import { withIronSessionApiRoute } from 'iron-session/next';
 import { NextApiRequest, NextApiResponse } from 'next';
 import { ironOptions } from '@/constants';
+import  { Contract, ethers } from "ethers";
+
+const provider = new ethers.providers.AlchemyProvider("rinkeby", process.env.NEXT_PUBLIC_ALCHEMY_KEY);
```

Comment:
i'm guessing this is set to rinkeby just for testing?

Reply:
Yup, we'll connect it to the real passport contracts eventually.

Reply:
Yup, just for testing.